### DISCUSSION TITLE
Search by table column name

### DISCUSSION THREAD

**Discussion by artgoldberg** [dbeaver/dbeaver#11170] [Q&A] [answered]
I'm a new DBeaver user who's loving it.

I'm working with a new database that contains many dozens of tables. I believe that it contains columns with a certain name. The [docs show](https://dbeaver.com/docs/wiki/DB-Metadata-Search/) a search that searches Table columns, but my Search -> DB Metadata doesn't offer Table columns. Can it be configured to search them?
![DBeaver search](https://user-images.githubusercontent.com/33882/106329786-0f633e00-6250-11eb-8ac4-7c3a63947ca8.png)

DBeaver 7.3.3.202101161839
Oracle DBMS

Thanks

**Comment by pierremorallet** [comment, marked as answer]
Unfortunately, this feature doesn't seem to be implemented for Oracle DB.
See issue #8684

**Comment by E1izabeth** [comment]
You can vote for the mentioned issue with 👍🏻 emoji under the issue description.

**Comment by TheMechanist902** [comment]
Hi,
I noticed this problem exists for MSSQL as well and found a possible workaround. I wrote down the details in [my answer in issue #8664](https://github.com/dbeaver/dbeaver/issues/8684#issuecomment-2040715823).
